C#/cs01/tests/13.cs

```diff
@@ -0,0 +1,59 @@
+using NUnit.Framework;
+using Brainchild.Curriculum.Solutions;
+using System;
+namespace Brainchild.Curriculum
+{
+    [TestFixture]
+    public class Test13
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+
+        [Test]
+        public void CheckingGenericClassStringStack()
+        {
+            Stack<string> stringstack = new Stack<string>();
+            stringstack.Push("abc");
+            stringstack.Push("abcd");
+            stringstack.Push("abcde");
+            stringstack.Pop();
+            string[] actual = stringstack.Output();
+            string[] expecting = new string[1000];
+            expecting[0] = "abc";
+            expecting[1] = "abcd";
+            Assert.AreEqual(actual, expecting);
```

Comment:
this test is wrong. iyou pushed three values and its only return two ?

Reply:
I did pushed 3 times and did a pop() also, so it will only return 2 values.

public void CheckingGenericClassStringStack()
        {
            Stack<string> stringstack = new Stack<string>();
            stringstack.Push("abc");
            stringstack.Push("abcd");
            stringstack.Push("abcde");
            stringstack.Pop();
            string[] actual = stringstack.Output();
            string[] expecting = new string[1000];
            expecting[0] = "abc";
            expecting[1] = "abcd";
            Assert.AreEqual(actual, expecting);
        }